package.json

```diff
@@ -21,104 +21,88 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@ember/ordered-set": "^1.0.0",
-    "amd-name-resolver": "0.0.7",
-    "babel-plugin-debug-macros": "^0.1.11",
-    "babel-plugin-ember-modules-api-polyfill": "^1.4.2",
+    "@ember/ordered-set": "^2.0.0",
     "babel-plugin-feature-flags": "^0.3.1",
     "babel-plugin-filter-imports": "^0.3.1",
-    "babel-plugin-transform-es2015-block-scoping": "^6.24.1",
+    "babel-plugin-transform-es2015-block-scoping": "^6.26.0",
     "babel6-plugin-strip-class-callcheck": "^6.0.0",
     "babel6-plugin-strip-heimdall": "^6.0.1",
-    "broccoli-debug": "^0.6.2",
-    "broccoli-file-creator": "^1.0.0",
+    "broccoli-debug": "^0.6.4",
+    "broccoli-file-creator": "^2.1.1",
     "broccoli-funnel": "^2.0.1",
-    "broccoli-merge-trees": "^2.0.0",
-    "broccoli-rollup": "^2.1.0",
+    "broccoli-merge-trees": "^3.0.0",
+    "broccoli-rollup": "^2.1.1",
     "calculate-cache-key-for-tree": "^1.1.0",
-    "chalk": "^1.1.1",
-    "ember-cli-babel": "^6.15.0",
+    "chalk": "^2.4.1",
```

Comment:
Major version bump due to dropping Node 4 support (a few internal APIs changed, but we are not using them)